test/integration/guest_user_test.rb

```diff
@@ -21,10 +21,10 @@ def setup
                             email: 'jwan622@example.com',
                             role: 0)
 
-    @item1 = Item.create(title: 'espresso', description: "this is black gold", price: 30000)
-    @category1 = item1.categories.create(name: 'Hot Beverages')
-    @item2 = Item.create(title: 'cold pressed coffee', price: 8000, description: "hipster nonsense")
-    @category2 = item2.categories.create(name: 'cold beverages')
+    @category1 = Category.create(name: 'Hot Beverages')
+    @category2 = Category.create(name: 'cold beverages')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.